apps/dialtone-documentation/docs/components/toast.md

```diff
@@ -68,9 +68,9 @@ htmlCode='
 vueCode='
 <dt-toast
   title="Title"
-  show="false"
-  important="false"
-  kind="Base"
+  show="showToast"
+  important="important"
+  kind="selectedKind"
```

Comment:
These are referring to variables right? so should be `:show`, `:important`, `:kind`.